HeatApp/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  createAppContainer, createBottomTabNavigator, createSwitchNavigator
+} from "react-navigation";
+
+import FavouritesScreen from './components/screens/FavouritesScreen';
+import MenuScreen from './components/screens/MenuScreen';
+import UserSettingsScreen from './components/screens/UserSettingsScreen';
+import StartSettingsScreen from './components/screens/StartSettingsScreen';
+
+const MainTabs = createBottomTabNavigator(
+  {
+    Favourite: FavouritesScreen,
+    Menu: MenuScreen,
+    Settings: UserSettingsScreen
+  },
+  {
+    initialRouteName : 'Menu',
+    tabBarOptions: {
+      activeTintColor: "#308312"
```

Comment:
Let's move color definition to some file with constants, so we can be consistent about our color scheme.

Reply:
No problem)